pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/VeryGoodOpenSource/mockingjay
 
 environment:
   sdk: ">=2.18.0 <3.0.0"
-  flutter: ">=3.7.3"
+  flutter: ">=3.13.0"
```

Comment:
As discussed async, we should verify if we need to bump this. Otherwise it's a breaking release

Reply:
Just checked and we don't! Just reverted